src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState, memo } from 'react';
+import ActionsArrowDownIcon from 'icons/react/ActionsArrowDownIcon';
+import ActionsArrowUpIcon from 'icons/react/ActionsArrowUpIcon';
+import { IconButton } from 'components';
+import HamburgerBarsIcon from 'icons/react/HamburgerBarsIcon';
+import DenyIcon from 'icons/react/DenyIcon';
+import CBILogo from './CBI_Logo_White.svg';
+import {
+  WhoWeServeDropdown,
+  HowWeHelpDropdown,
+  WhatWeOfferDropdown,
+  ResourcesDropdown,
+} from './constants';
+import NavBarDropdown from './NavbarDropdown';
+
+const NAV_LIST = 'navList';
+const WHO_WE_SERVE = 'whoWeServe';
+const HOW_WE_HELP = 'howWeHelp';
+const WHAT_WE_OFFER = 'whatWeOffer';
+const RESOURCES = 'resources';
+
+const DEFAULT_STATE: { [key: string]: boolean } = {
+  [NAV_LIST]: false,
+  [WHO_WE_SERVE]: false,
+  [HOW_WE_HELP]: false,
+  [WHAT_WE_OFFER]: false,
+  [RESOURCES]: false,
+};
+
+export type NavBarProps = {
+  domain: string;
+  utmContext: string;
+};
+
+const NavBar = ({ domain, utmContext }: NavBarProps): JSX.Element => {
+  const [showClass, setShowClass] = useState(DEFAULT_STATE);
+
+  const handleClick = (sectionName: string): void => {
+    setShowClass({ ...showClass, [sectionName]: !showClass[sectionName] });
```

Comment:
If you new state depends on your previous state it is better to use the callback to set the new state to avoid overwriting changes that are executed at the same time

https://reactjs.org/docs/hooks-reference.html#functional-updates